AllJoyn/Platform/Bridge/BridgeCore/AllJoynHelper.cpp

```diff
@@ -1339,14 +1460,21 @@ std::shared_ptr<IAdapterValue> AllJoynHelper::GetDefaultAdapterValue(_In_ const
         case 's':
             value = std::vector<std::string>{};
             break;
+
+        case '{':
+            if (strcmp(signature, "{ss}") == 0)
+            {
+                value = std::unordered_map<std::string, std::string>{};
+            }
+            break;
         }
     }
 
     if (value.Type() != PropertyType::Empty)
     {
-		AdapterValue* aval = new AdapterValue(std::string(), value);
-		std::shared_ptr<AdapterValue> sval(aval);
-		return sval;
+        AdapterValue* aval = new AdapterValue(std::string(), value);
```

Comment:
I didn't actually touch this block code. Note the difference is whitespace only: my IDE cleaned up inconsistent tabs vs spaces.

Generally it's not my intention to clean up existing code except where I'm refactoring to support my change.